src/components/doctorInfo/AddDoctor.js

```diff
@@ -0,0 +1,191 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { FaXmark } from 'react-icons/fa6';
+import {
+  Form, Button, Row, Col, Modal,
+} from 'react-bootstrap';
+import { addDoctor } from '../../features/doctorSlice';
+
+const initialAddressData = {
+  street: '',
+  city: '',
+  state: '',
+  zip_code: '',
+};
+
+const initialFormData = {
+  name: '',
+  age: '',
+  email: '',
```

Comment:
Excellent work 🎉🎉😍thus far! However, based on our Kanban board, it appears that these particular parts should not be handled individually. Please remove this code so that the designated person responsible for this task can take care of it. 🤔🤔🤔